src/lib.rs

```diff
@@ -1,14 +1,13 @@
-//! A wrapper around the uchardet library.  Detects character encodings.
+//! A wrapper around the uchardet library. Detects character encodings.
 //!
 //! Note that the underlying implemention is written in C and C++, and I'm
 //! not aware of any security audits which have been performed against it.
 //!
 //! ```
 //! use uchardet::detect_encoding_name;
 //!
-//! assert_eq!(Some("windows-1252".to_string()),
-//!            detect_encoding_name(&[0x66u8, 0x72, 0x61, 0x6e, 0xe7,
-//!                                 0x61, 0x69, 0x73]).unwrap());
```

Comment:
Does the detector still detect `"windows-1252"` and output that string? I notice that this PR removes several tests related to this encoding.

For my use case, I encounter a lot of input data in `"windows-1252"` (including older subtitle data for European films), and I want to make sure the detector can still detect that format correctly.